content/pt-br/docs/tasks/access-application-cluster/communicate-containers-same-pod-shared-volume.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Comunicação entre contêineres no mesmo pod usando um volume compartilhado
+content_type: task
+weight: 110
+---
+
+<!-- overview -->
+
+Esta página mostra como usar um Volume para se comunicar entre dois Containers rodando
```

Comment:
"Containers" should be translated in this sentence ("contêineres"). The capital C does not need to be kept.

Reply:
Maybe we could have: "Esta página mostra como usar um Volume para realizar a comunicacão entre dois Containers rodando..."